os-server/src/main/java/app/openschool/user/UserServiceImpl.java

```diff
@@ -38,6 +39,7 @@ public class UserServiceImpl implements UserService {
   private final CourseStatusRepository courseStatusRepository;
   private final ModuleStatusRepository moduleStatusRepository;
   private final ModuleItemStatusRepository moduleItemStatusRepository;
+  private static final Pattern CLEAR_PATTERN = Pattern.compile("\\s+");
```

Comment:
what this pattern does ?

Reply:
removes spaces between two words if there are more than one